src/arena.rs

```diff
@@ -203,15 +218,24 @@ pub use self::sys::SysAllocator;
 /// Naïve memory-safe implementation of `Arena`.
 pub mod checked {
     use super::*;
-    use std::collections::HashMap;
+    #[cfg(all(not(feature = "std"), feature = "hashbrown"))]
```

Comment:
I prefer not having `hashbrown` support at all. The performance loss due to using `BTreeMap` is acceptable because `CheckedArena` is:

https://github.com/yvt/xalloc-rs/blob/81d5a8efea48f693ae4d953d736478d3dc8656db/src/arena.rs#L211

Reply:
I'm not familiar with this algorithm, but will changing to `BTreeMap` change the time complexity of `TLSF`?

Reply:
It only affects [`SafeTlsf`](https://docs.rs/xalloc/0.2.7/xalloc/tlsf/type.SafeTlsf.html).